lib/ctsquery-builder.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2021 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const bldrgen  = require('./plan-builder-generated.js');
+
+    /**
+     * A helper for building the definition of a rows query. The helper is
```

Comment:
In this context, it would be better to delete the word "rows" because we're getting documents instead of rows

Reply:
Thanks Erik.! Included the change.